web/themes/custom/pineapple/sass/layouts/nodes/_article-teaser-page.scss

```diff
@@ -0,0 +1,51 @@
+.views-row {
+  margin-bottom: 16px;
+}
```

Comment:
Some issues with this styling:

- Currently this styling effects every single views row on the site (every view that exists and will exist will have a views row) you need to make this more specific to your particular view (before doing this, read the rest of the comment)

- This styling adds margin bottom to every views row, no matter what. since these classes are repeated this would be a good time to use the **owl** class on the container of the views row, so that the styling would be added somewhat "dynamically"

- A solution to achieve both of the comments above, would be to create a custom template for your particular view and then add the owl class and that way you can remove this styling and not have to write new css

Reply:
done